app/src/pages/Home/operations.graphql

```diff
@@ -20,7 +20,6 @@ mutation BecomeHost($gameId: uuid!, $playerId: uuid!) {
     join_code
     starting_letter
     seconds_per_turn
-    num_entries_per_player
```

Comment:
This was actually inadvertently removed; I'll add it back. But I was a little confused by this query in particular – would we want to expose `allow_card_skips` here as well?

Reply:
We actually can get rid of join_code, starting_letter, seconds_per_turn, and num_entries_per_player. None of the values are used in the call site!

https://github.com/avimoondra/fishbowl/blob/master/app/src/pages/Home/index.tsx#L71

I at least keep id in all queries/mutations just bc of prior experience running into issues when not having those w/apollo cache. in this case, we'd probably not even need that.

Feel free to do that now, or in a subsequent pr (preferred)

Reply:
also if you remove something from a query/mutaiton and it doesn't generate a type issue... large majority of the time we are good (only useLazyQuery i think is least typed 😢)